docs/src/main/sphinx/installation/cli.rst

```diff
@@ -405,6 +418,25 @@ Press :kbd:`→` to accept the suggestion and replace the current command line
 buffer. Press :kbd:`Ctrl+→` (:kbd:`Option+→` on Mac) to accept only the next
 keyword. Continue typing to reject the suggestion.
 
+Configuration file
+------------------
+
+The CLI can read default values for all options from a file. It uses the first
+file found from the ordered list of locations:
```

Comment:
@nineinchnick What is the meaning of first file in ordered list of locations? Can we specify multiple files? If yes, how do we use multiple files? 

I am not sure if this can be helpful but when specifying boolean properties it might be helpful to mention in the doc the boolean properties can be written like 

`krb5-disable-remote-service-hostname-canonicalization=true`


Additional question, if I have multiple authentication enabled like Kerberos, Password, Can I use multiple files with different list of arguments and how do I activate them?

Reply:
Excellent comment, thanks! I extracted this change to [a separate PR](https://github.com/trinodb/trino/pull/12815) and applied these suggestions, that is explained how to use a custom config and described boolean values.